schemas/data/datasource.schema.json

```diff
@@ -46,5 +46,5 @@
       "$ref": "#/definitions/datasource"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
#203 is still open 👎

Reply:
#203 is a documentation request, we should address for 1.0 but not hold up the 0.9 milestone.

Reply:
As outlined in the discussions around #240 – I don't think we are just there yet.

Reply:
👎 The Tags issue needs to be resolve since we can't have xdm:Tags in both assets and here and they mean something different.